test/test_nn.py

```diff
@@ -11160,10 +11159,16 @@ def test_hardswish_grad_corner(self, device, dtype):
         cpu_input = cpu_input * 3
         cpu_input.requires_grad = True
         fwd_result = m(cpu_input)
-        grad = torch.ones_like(fwd_result)
-        fwd_result.backward(grad)
+        fwd_result.backward(torch.ones_like(fwd_result))
         ref = torch.ones(shape, device=device, dtype=dtype)
-        ref.fill_(1.5)
+        self.assertEqual(cpu_input.grad, ref)
+
+        cpu_input = torch.ones(shape, device=device, dtype=dtype)
+        cpu_input = cpu_input * -3
```

Comment:
Since the vectorized version is altered, we should add tests with larger shapes.

Reply:
hmm the current shapes are (1, 9, 9, 1) - maybe sufficiently big enough?

Reply:
Yeah, it should be fine, indeed.